README.md

```diff
@@ -6,11 +6,78 @@ implemented in pure Go, i.e. does not wrap libnftnl.
 
 This is not an official Google product.
 
-## Breaking changes
+## Alpha status
+
+This package is in early stages, and only implements a subset of nftables
+features. While the developers intend to keep interfaces & function signatures
+backwards-compatible, no guarantees are made; bugs or any unexpected
+structuring of nftables features may result in breaking changes.
+
+## Usage
+
+Issue commands to mutate or read nftables state. Commands that mutate state
+(eg: `AddTable`, `AddChain`, `AddSet`, `AddRule`) are queued until `Flush`
+is called.
+
+### Expressions
+
+The following expressions are implemented for use in rule logic:
+
+TODO
+
+### Examples
+
+#### Drop outgoing packets to 1.2.3.4
+
+```go
```

Comment:
Could we link to a compiled example in godoc instead? That way, it’ll always be up to date.

Reply:
I'm not a huge fan of long examples in godoc, could we have an 'examples' folder with a bunch instead?

Alternatively, I'm writing up heaps with this package on [my](https://tomdnetto.net/post/linux_networking_from_go_nftables) [blog](https://tomdnetto.net/post/advanced_nftables_with_go). I don't think it would be right to link to my blog, but maybe we could put equivalent content in a wiki on this repo?

Reply:
An examples directory works, too, but please make them be testable.

I don’t mind linking to your blog (in addition). That might be a more pragmatic alternative than wikis, which only make sense for larger projects where there’s enough motivation around to keep them up-to-date.